### DISCUSSION TITLE
Update Open Library through Openreads

### DISCUSSION THREAD

**Discussion by Grebneh** [mateusz-bak/openreads#723] [Ideas]
**Describe the solution you'd like**
It would be nice if adding a book to the library of Openreads would automatically add the book to Open Library.

**Comment by pandalamp** [comment]
I support adding a link to Open Library, but this suggestion goes against the spirit of a **privacy oriented** app.

Besides, Open Library already has huge organizational problems that would make it extremely difficult to parse a title added here, likely without ISBN because what % of users is typing that accurately on mobile if at all, as its own unique edition of which work, manage conflicts with existing editions... Any users that could actually use such a feature are just as able to add the titles into Open Library themselves.

**Comment by justeepin** [comment]
How about a self-hostable version of OpenReads to run on a NAS for the client apps to sync to?

**Comment by mohammadrafigh** [comment]
It would be great if we could login with Open Library account and sync books, status and ratings.
